android/app/src/androidTest/java/com/salesforce/samples/smartsyncexplorerreactnative/ReactNativeTestRunner.java

```diff
@@ -0,0 +1,50 @@
+package com.salesforce.samples.smartsyncexplorerreactnative;
```

Comment:
Can we use the standard test runner we use [here](https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/dev/libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/util/test/JUnitReportTestRunner.java) for all our tests, instead of this test runner?